scipy/optimize/minpack.py

```diff
@@ -493,8 +498,23 @@ def curve_fit(f, xdata, ydata, p0=None, sigma=None, absolute_sigma=False,
         If True, check that the input arrays do not contain nans of infs,
         and raise a ValueError if they do. Setting this parameter to
         False may silently produce nonsensical results if the input arrays
-        do contain nans.
-        Default is True.
+        do contain nans. Default is True.
+    bounds : 2-tuple of array_like, optional
+        Lower and upper bounds on independent variables. Default is None
+        (no bounds). If provided, each element of the tuple must be either an
+        array with the length equal to the number of parameters, or a scalar
+        (in which case the bound is taken to be the same for all parameters.)
+        .. versionadded:: 0.17
```

Comment:
you do in the Notes section but not at the end of a parameter description, so this will be ok. not sure if it was designed like that on purpose....